openpathsampling/tests/test_integration_tools.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from openpathsampling.integration_tools import _chain_import
+
+@pytest.mark.parametrize('modules', [
+    ('foo', 'os.path'),
+    ('os.path', 'foo')
+])
+def test_chain_import(modules):
+    import os.path
+    mod = _chain_import(*modules)
+    assert mod is os.path
+
+def test_chain_import_error():
+    with pytest.raises(ImportError):
```

Comment:
Do you mind testing "last error" behavior? 
```suggestion
    with pytest.raises(ImportError, match="bar"):
```